README.md

```diff
@@ -1,2 +1,52 @@
-# vc-powershell-template
-Powershell script for creating Virto Commerce module solution
+# VC module template
+
+## Overview
+
+This script is a lite version of [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates)
+The script creates a solution template for the VirtoCommerce platform new module. The script automatically creates all the necessary projects:
```

Comment:
It generates a new Visual Studio solution for a module in Virto Commerce Platform.

Reply:
fixed